databases/week1/sql-homework-week1.sql

```diff
@@ -0,0 +1,85 @@
+SELECT
+  count(*)
+FROM
+  task;
+--task1--
+SELECT
+  due_date
+FROM
+  task
+WHERE
+  due_date IS NULL;
+--task2--
+SELECT
+  title,
+  name
+FROM
+  status
+  JOIN task ON status.id = task.status_id
+WHERE
+  name = 'done';
+--task3--
+SELECT
+  title,
+  name
+FROM
+  status
+  JOIN task ON status.id = task.status_id
+WHERE
+  name != 'done';
+--task4--
+SELECT
+  title,
+  created
+FROM
+  task
+ORDER BY
+  created DESC;
+--task5--
+SELECT
+  title,
+  created
+FROM
+  task
+ORDER BY
+  created DESC
+LIMIT
+  1;
+--task6--
+SELECT
+  title,
```

Comment:
This works 👍 
But keep in mind that the task requires just **the title and due date**, not the description. 
Sometimes, less is more 😄